mappings/net/minecraft/client/gui/GuiContainer.mapping

```diff
@@ -15,6 +15,11 @@ CLASS none/bhc net/minecraft/client/gui/GuiContainer
 		ARG 2 mouseY
 	METHOD a draw (IIF)V
 	METHOD a handleMouseClick (III)V
+	METHOD a mouseClick (Lnone/act;IILnone/abz;)V
+		ARG 0 slot
+		ARG 1 slotID
```

Comment:
Should this be `invSlot`? That's what I saw others using and made sure to do myself. Perhaps `invSlotId` or `invIndex`, as in the value you'd have to use to get the stack through the inventoy methods. Either way we should keep them consistent, so a simple search & replace can update all of them at once.